### ISSUE TITLE
"you know what maybe the DVM is ready for the people too"

### ISSUE THREAD

**Issue by oraclown** [tellor-io/disputable-values-monitor#30]
What are the main things that need to happen w/ this repo to make it ready for public consumption?

We could have a two-stage release, 1) just link this repo to the public and say "git clone", then later 2) package it and upload a first version to PyPI for people to install.

To do:
- [ ] Do a clean install on a fresh env and make sure the setup instructions work
- [ ] Verify the README is accurate regarding the features/behavior of the tool
- [ ] What's the longest time an instance of the disputable-values-monitor has been running, @spuddyminer ?
- [ ] ...

**Comment by spuddyminer** [comment]
If the APIs behave (no rate limits) and the reporters submit for established query types / feeds, the disputable-values-monitor can run for a long time.

The longest error-free "DVM" instance I've seen lasted about 72 hours while monitoring mainnet Ethereum and Polygon. When it's listening on the testnets (Rinkeby and Mumbai) It is less stable, and will error out after a few hours if we're testing new kinds of query types / feeds.

**Comment by oraclown** [comment]
> If the APIs behave (no rate limits) and the reporters submit for established query types / feeds, the disputable-values-monitor can run for a long time.
> 
> The longest error-free "DVM" instance I've seen lasted about 72 hours while monitoring mainnet Ethereum and Polygon. When it's listening on the testnets (Rinkeby and Mumbai) It is less stable, and will usually error out within 24 hours of starting.
> 
> Will do a clean install and comment with the results.

What happens if the APIs get rate-limited?
What happens if reporters report for unsupported query types?
Does the disputable-values-monitor stop running if either of those^ happen? If, so we need to make issues & fix before release.

Any errors that we don't have issues for / aren't working on?

**Comment by spuddyminer** [comment]
Yes, the disputable-values-monitor sometimes stops running when there are hiccups from APIs and new types of submissions. As of this morning these issues still need fixes:

https://github.com/tellor-io/disputable-values-monitor/issues/20
https://github.com/tellor-io/disputable-values-monitor/issues/18

**Comment by tallywiesenberg** [comment]
Closed #20 as it has been caught by a more recent version
